data/Person.csv

```diff
@@ -270,8 +270,8 @@ P0150,Link,Perry,en,male,NULL,NULL,NULL,NULL,1944,,NULL,NULL,,NULL,NULL,https://
 P0150,林,培瑞,zh,male,NULL,NULL,NULL,NULL,1944,,NULL,NULL,,NULL,NULL,https://en.wikipedia.org/wiki/Perry_Link,,,,,,2017-09-22,MG,2017-12-23,MG
 P0151,Camus,Albert,fr,male,NULL,NULL,NULL,NULL,1913,1960,NULL,NULL,PL0073,NULL,NULL,https://de.wikipedia.org/wiki/Albert_Camus,,,,,,2017-09-22,MG,2017-12-23,MG
 P0151,加缪,阿尔贝,zh,male,NULL,NULL,NULL,NULL,1913,1960,NULL,NULL,PL0073,NULL,NULL,https://zh.wikipedia.org/wiki/%E9%98%BF%E5%B0%94%E8%B4%9D%C2%B7%E5%8A%A0%E7%BC%AA,,,,,,2017-09-22,MG,2017-12-23,MG
-P0152,Cao,Xueqin,en,male,NULL,NULL,NULL,NULL,1715-1724,1762-1763,,,,NULL,NULL,https://de.wikipedia.org/wiki/Der_Traum_der_Roten_Kammer,,,,,,2017-09-22,MG,2017-11-06,OS
-P0152,曹,雪芹,zh,male,NULL,NULL,NULL,NULL,1715-1724,1762-1763,,,,NULL,NULL,https://zh.wikipedia.org/wiki/%E6%9B%B9%E9%9B%AA%E8%8A%B9,,,,,,2017-09-22,MG,2017-11-06,OS
+P0152,Cao,Xueqin,en,male,NULL,NULL,NULL,NULL,[1715-1724],[1762-1763],,,,NULL,NULL,https://de.wikipedia.org/wiki/Der_Traum_der_Roten_Kammer,,,,,,2017-09-22,MG,2017-11-06,OS
+P0152,曹,雪芹,zh,male,NULL,NULL,NULL,NULL,[1715-1724],[1762-1763],,,,NULL,NULL,https://zh.wikipedia.org/wiki/%E6%9B%B9%E9%9B%AA%E8%8A%B9,,,,,,2017-09-22,MG,2017-11-06,OS
```

Comment:
Please change to EDTF `[1715,1724]` and `[1762,1763]` see [specs](https://www.loc.gov/standards/datetime/ISO_DIS%208601-2.pdf) p. 8, 10, 14 for what they mean.